tests/unit/conftest.py

```diff
@@ -28,66 +28,63 @@
 from gordon import interfaces
 
 
-PLUGIN_NAMES = ['event_consumer', 'enricher', 'publisher', 'runnable']
+PLUGIN_NAMES = [
+    'event_consumer',
+    'enricher',
+    'publisher',
+    'cleaner',
```

Comment:
I know you don't like the idea of the event consumer doing the cleaning up, and I'm not sure the interface definition technically separates that out, but I'd _really_ like to talk that part through before we start doing it, even eluding to it in tests. The main reason is (unfortunately) the gcp event consumer itself is closely tied with the cleanup phase as the consuming of a message is closely integrated with the ack'ing/cleaning up of the message. We'd need to think about how to untangle that first.

Reply:
The reason I broke it out was because it was breaking the phase attribute defined in the generic plugin interface and complicated the routing logic a bit.